sdk/samples/iot/provisioning/README.md

```diff
@@ -44,8 +44,28 @@ Further background on the Azure IoT Client library and key concepts are explaine
 * Have Microsoft [VCPKG](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/) installed.  Use the directions [here](https://github.com/Azure/azure-sdk-for-c#development-environment) to download VCPKG and install Paho MQTT.
 
 ## Getting Started
+### Environment Variables
+Samples use environment variables for a variety of purposes, including filepaths and connection parameters. Please keep in mind, every time a new terminal is opened, the environment variables will have to be reset.  Setting a variable will take the following form:
```

Comment:
nit: We seem to have a bunch of double white space `  `. Find and remove them.
```suggestion
Samples use environment variables for a variety of purposes, including file paths and connection parameters. Please keep in mind, every time a new terminal is opened, the environment variables will have to be reset. Setting a variable will take the following form:
```

Reply:
I did some reading on this, and filepath as one word in this context is completely acceptable.  https://english.stackexchange.com/questions/428215/filepath-or-file-path#:~:text=A%20filepath%20is%20an%20entity%2C%20independent%20of%20any,not%20specifying%20what%20particular%20file%20we%27re%20talking%20about%29.#:~:text=A%20filepath%20is%20an%20entity%2C%20independent%20of%20any,not%20specifying%20what%20particular%20file%20we%27re%20talking%20about%29.#:~:text=A%20filepath%20is%20an%20entity%2C%20independent%20of%20any,not%20specifying%20what%20particular%20file%20we%27re%20talking%20about%29.